Solutions/Marain.Instance.Deployment/Deploy-MarainInstanceInfrastructure.ps1

```diff
@@ -441,13 +461,18 @@ class MarainServiceDeploymentContext {
         [string]$AppServiceName)
     {
         $WebApp = Get-AzWebApp -Name $AppServiceName
+        if (-not $WebApp) {
+            Write-Error "Did not fine web app $AppServiceName"
```

Comment:
Should that be "did not find"?

Reply:
Looks fine to me.

I'm here all week.

Pushed fix.